website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -95,6 +95,7 @@ A `addon_profile` block supports the following:
 
 * `http_application_routing` - (Optional) A `http_application_routing` block.
 * `oms_agent` - (Optional) A `oms_agent` block. For more details, please visit [How to onboard Azure Monitor for containers](https://docs.microsoft.com/en-us/azure/monitoring/monitoring-container-insights-onboard).
+* `aci_connector_linux` - (Optional) A `aci_connector_linux` block. For more details, please visit [Create and configure an AKS cluster to use virtual nodes](https://docs.microsoft.com/en-us/azure/aks/virtual-nodes-portal).
```

Comment:
_minor_ can we put this above the other two items (so this is sorted alphabetically?)

Reply:
Moved it to the top.